holmes/plugins/toolsets/datadog/datadog_metrics_instructions.jinja2

```diff
@@ -24,3 +31,19 @@ When investigating metrics-related issues:
 - CPU investigation: First list metrics with `tag_filter:kube_node_name:nodename`, then query specific metrics
 - Memory issues: Look for `system.mem.*` or `kubernetes.memory.*` metrics
 - Container metrics: Filter by pod/container tags
+
+
+# Handling queries results
+* ALWAYS embed the execution results into your answer
+* You only need to embed the partial result in your response. Include the "tool_name" and "random_key". For example: << {"type": "promql", "tool_name": "execute_prometheus_range_query", "random_key": "92jf2hf"} >>
+* Post processing will parse your response, re-run the query from the tool output and create a chart visible to the user
+* The toolcall will return no data to you. That is expected. You MUST however ensure that the query is successful.
+* ALWAYS embed the execution results into your answer
```

Comment:
Duplicate, already on the first row of this section

Reply:
fixed